ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Idpbuilder
+If any organization would like to get added or removed please make a pull request by following these guidelines:
+
+*  For the organization column make the organization name a link that points to the orgs website if available. Please don't include your organization's logo or other trademarked material.
+* Add a short description on how your organization is benefiting from using idpbuilder,  is encourage to also include link to a public (blog post, video, slides, git repo, etc.) that shows or mentions how Idpbuilder is used
```

Comment:
```suggestion
* Add a short description on how your organization is benefiting from using idpbuilder. We encourage to also include a link to a public (blog post, video, slides, git repo, etc.) that shows or mentions how Idpbuilder is used
```

Reply:
this looks good apply the suggested changes